menu/additional_apps.sh

```diff
@@ -125,7 +125,7 @@ to finish the setup once this script is done." "$SUBTITLE"
     ;;&
     *"Talk"*)
         print_text_in_color "$ICyan" "Downloading the Talk script..."
-        run_script APP talk
+        desec_app_tls_menu "DESEC talk_desec" "APP talk"
```

Comment:
honestly, I find it a bit strange that this will now always go through this menu. It will be yet another step

Reply:
Do you have a better proposal?

Reply:
ping @szaimen

Reply:
Can we maybe check for the dessec folder and only show that menu if dessec is already configured?

Reply:
Hmm, should work.

Reply:
Thought about this, what about users that doesn't have deSEC installed and wants to use it?